tinymce-angular-component/src/main/ts/editor/Events.ts

```diff
@@ -11,6 +11,9 @@ export class Events {
   @Output() public onBeforePaste: EventEmitter<EventObj<ClipboardEvent>> = new EventEmitter();
   @Output() public onBlur: EventEmitter<EventObj<FocusEvent>> = new EventEmitter();
   @Output() public onClick: EventEmitter<EventObj<MouseEvent>> = new EventEmitter();
+  @Output() public onCompositionEnd: EventEmitter<EventObj<CompositionEvent>> = new EventEmitter();
+  @Output() public onCompositionStart: EventEmitter<EventObj<CompositionEvent>> = new EventEmitter();
+  @Output() public onCompositionUpdate: EventEmitter<EventObj<CompositionEvent>> = new EventEmitter();
```

Comment:
Should these values be included in the `validEvents` below?

Reply:
Oh, true. Thank you